packages/components/src/navigator/test/index.tsx

```diff
@@ -396,8 +428,6 @@ describe( 'Navigator', () => {
 	} );
 
 	it( 'should escape the value of the `path` prop', async () => {
-		const user = userEvent.setup();
```

Comment:
Moved part of this test to a new test (`should restore focus correctly even when the `path` needs to be escaped`). Reviewing commit-by-commit should make this part easier to review.